.github/workflows/build.yml

```diff
@@ -41,7 +41,8 @@ jobs:
           - "spec/serializers"
           - "spec/system/admin/[a-o0-9]*"
           - "spec/system/admin/[p-z]*"
-          - "spec/system/consumer"
+          - "spec/system/consumer/[a-o0-9]*"
+          - "spec/system/consumer/[p-z]*"
```

Comment:
Is [this syntax](https://github.com/openfoodfoundation/openfoodnetwork/issues/9676#issuecomment-1249004958) preferred here instead @jibees?

Reply:
Nop, I don't think there is any difference. Go ahead with this one.